flexget/plugins/output/notify.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.event import event
+
+log = logging.getLogger('notify')
+
+HANDLED_PHASES = ['start', 'input', 'filter', 'output', 'exit']
+
+
+class Notify(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'to': {'type': 'array', 'items':
+                {'allOf': [
+                    {'$ref': '/schema/plugins?group=notifiers'},
+                    {'maxProperties': 1,
+                     'error_maxProperties': 'Plugin options within notify plugin must be indented '
+                                            '2 more spaces than the first letter of the plugin name.',
+                     'minProperties': 1}]}},
+            'scope': {'type': 'string', 'enum': ['task', 'entries']},
+            'what': one_or_more({'type': 'string', 'enum': ['entries', 'accepted', 'rejected', 'failed', 'undecided']}),
+            'phase': one_or_more({'type': 'string', 'enum': HANDLED_PHASES, 'default': 'output'})
+        },
+        'required': ['to'],
+        'additionalProperties': False
+    }
+
+    @staticmethod
+    def prepare_config(config):
+        if not isinstance(config['phase'], list):
+            config['phase'] = [config['phase']]
+        if not isinstance(config['what'], list):
+            config['what'] = [config['what']]
+        config.setdefault('what', ['accepted'])
+        config.setdefault('scope', 'entries')
+        config.setdefault('phase', ['output'])
```

Comment:
Can you think of any reason we'd need alternate phases? If not, I'd skip this as a config option and add it in later if we think of some use.